### PR TITLE
README.md updated

### PR DESCRIPTION
**Pull request by adibns**

### CODE REVIEW

**Review by charlessuresh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,15 +15,47 @@ $ pip install -i https://test.pypi.org/simple/ datascience_eda
 
 ## Features
 
-- TODO
+- The function `get_clustering_default_hyperparameters` creates a dictionary listing default hyperparameters for K-Means and DBSCAN clustering and returns a dictionary with key being name of the clustering algorithm and the value being a dictionary of hyperameters' names and values
```
Comment (lephanthuymai) [inline README.md]: this is a helper function, we do not need to document it.

**Comment by lephanthuymai** [comment]
Hi Aditya,
Please review the requirements of README:

- a summary paragraph that describes the project at a high level (MISSING)
- a bulleted list of the functions (and datasets if applicable) that will be included in the package (this should be a 1-2 sentence description for each function/dataset)
- a paragraph describing where your packages fit into the Python ecosystem (are there any other Python packages that have the same/similar functionality? Provide links to any that do. If none exist, then clearly state this as well). (MISSING)
Thanks.

File: README.md [hunk 2]
```diff
@@ -15,15 +15,47 @@ $ pip install -i https://test.pypi.org/simple/ datascience_eda
...
+yellowbrick 1.3.post1
+spacy 3.0.3
+nltk 3.5
+seaborn 0.11.1
 
 ## Usage
 
-- TODO
+`get_clustering_default_hyperparameters`
+    >>> hyper_dict = get_clustering_default_hyperparameters()
```
Comment (lephanthuymai) [inline README.md]: Usage section is for how to use the package, not how to use the functions.

**Comment by adibns** [comment]
@lephanthuymai Please make changes as per your wish. From next milestone I think we have to comprehensively go through your expectations.

**Comment by lephanthuymai** [comment]
Hi Aditya, these are not my expectations, I copied them from the milestone requirements https://pages.github.ubc.ca/MDS-2020-21/DSCI_524_collab-sw-dev_students/materials/assignments/milestone1.html#. I also copied it into the Github issue assigned to you: https://github.com/UBC-MDS/datascience_eda/issues/6
I would hope every member should read the Milestone scope and the issues assigned to them as we are equally responsible for completing what is required for this project.
Hope this helps.
Thanks.

**Pull request closed without merging**